.gitignore

```diff
@@ -8,3 +8,4 @@ error.html
 
 # Ignore local configuration
 /phpunit.xml
+.idea
```

Comment:
Could you move this to a global gitiginore and leave a newline character at the end?